paper/paper.md

```diff
@@ -79,7 +79,7 @@ download URLs) by downloading files from one or more remote servers and storing
 them in a local data cache.
 Pooch is written in pure Python and has minimal dependencies.
 It can be easily installed from the Python Package Index (PyPI) and conda-forge
-on a wide range of Python versions: 2.7 (up to Pooch 0.6.0) and from 3.5 to 3.8.
+on a wide range of Python versions: 2.7 (up to Pooch 0.6.0) and from 3.6 to 3.8.
```

Comment:
Wasn't sure whether to update this. Should it paper be kept up to date, in case a new edition is published at some point?

Reply:
Not really since we would likely need to do a big rewrite if we ever go for a second paper. There would need to be significant changes to Pooch for that to happen.